packages/mobile/src/components/oauth/OAuth.tsx

```diff
@@ -180,8 +174,6 @@ const OAuth = () => {
   const dispatch = useDispatch()
   const url = useSelector(getUrl)
   const isOpen = useSelector(getIsOpen)
-  const messageId = useSelector(getMessageId)
```

Comment:
I think we still need the messages in this file bc this is actually a different webview, specifcally for oauth